links.html

```diff
@@ -54,5 +54,6 @@ <h1>Useful Links</h1>
     </footer>
 
         <a href="#top" title="Back to top">◤</a>
+    </div>
```

Comment:
Cat ran across your keyboard at an inopportune time?

Reply:
I actually have no idea why that was there, I'm so sorry I missed it.